rasa/core/featurizers/single_state_featurizer.py

```diff
@@ -5,7 +5,9 @@
 from collections import defaultdict
 
 import rasa.shared.utils.io
-from rasa.nlu.constants import TOKENS_NAMES
+from rasa.nlu.classifiers.diet_classifier import EntityTagSpec
```

Comment:
should this be moved to a more neutral location?

Reply:
good point, I thought about it, but then thought meh 😅

Reply:
moved to `extractor.py`